### ISSUE TITLE
NerdTREE shortcuts not working / inconsistent

### ISSUE THREAD

**Issue by Robert-M-Muench** [jordwalke/VimBox#51]
I'm struggling with some shortcuts not working:

Works: ⌘+\   Toggle File Explorer (Docked NERDTree)
Doesn't work: ⌘+e  Toggle sidebar file explorer

Or are these two different things? The \ char is not very fortunate on a German keyboard. Is there a place where I can change it? Or are there are keyboard layout specific shortcuts?

**Comment by jordwalke** [comment]
I changed this to match Nuclide's (And Atom's) default behavior. I'd be happy to make it configurable - I really want a better way to configure keybindings. Something like VSCode's (or Atom's) keybinding configuration. It allows the user to override the keybindings before they even take place. The problem with vim, is that if I provide a default set of keybindings, you can then configure keybindings in your overlap on top of VimBox, but it's not the same as if the defaults were never configured. We need a default keybinding .json file that is merged with user customizations - before we even begin to apply the keybindings, so that none of the defaults even take place. If you'd be interested in helping out, I'd greatly appreciate it (or perhaps there's an existing plugin for managing keybindings in this manner that we can bring into VimBox).

**Comment by Robert-M-Muench** [comment]
> VimEnter: After doing all the startup stuff, including loading .vimrc files, executing the "-c cmd" arguments, creating all windows and loading the buffers in them.
So, anything you put in a VimEnter auto command gets run after Vim is ready. 

Not sure I'm understanding this. I think we need to change the keybindings after the plugins loaded because otherwise, we need to change the default keybinding on a plugin base. Doesn't sound very stable nor straightforward to me.

Maybe VimEnter (=after doing all the startup stuff, including loading .vimrc files, executing the "-c cmd" arguments, creating all windows and loading the buffers in them.) can help. So, anything you put in a VimEnter auto command gets run after Vim is ready.

**Comment by jordwalke** [comment]
Yeah, but how do users configure these keybindings such that their overlaid keybindings are executed *instead* of VimBox's defaults? I think there needs to be a configuration file like `.json` that is merged before any keybindings are executed.

**Comment by Robert-M-Muench** [comment]
I'm having the problem that ⌘+\ doesn't work anymore. Any idea how to track it down?
